static/js/impala/suggestions.js

```diff
@@ -130,12 +130,14 @@ $.fn.searchSuggestions = function($results, processCallback, searchType) {
             $results.filter('.visible').removeClass('visible');
             return;
         }
+        var urlVal = encodeURIComponent($self.val());
```

Comment:
That function escapes `&`, `>`, `<`, `'` and `"`. But almost all of those symbols are already processed by `encodeURIComponent`. The only exception is ­- `'`.

Query string must be percent-encoded [1]. And `encodeURIComponent` does it. `escape_` makes another replace and brakes strings with `'` in them. It adds ampersand, which is reserved symbol in query string.

[1] https://en.wikipedia.org/wiki/Percent-encoding